### PR TITLE
apis: generate homegraph api

### LINKED ISSUE #1806: generate binding for the homegraph api
The `homegraph` API is now listed under the discovery directory:
https://www.googleapis.com/discovery/v1/apis/

It would be nice to have generated bindings for it so that the smarthome library (https://github.com/actions-on-google/actions-on-google-nodejs/blob/master/src/service/smarthome/smarthome.ts) can depends on it rather than handcrafting a client.

### PR DESCRIPTION
**Pull request by proppy**
- run the generate
- clean up the diff to only include homegraph changes

Let me know if you'd prefer to update all the generate API as part of the bigger CL instead.

Fixes #1806

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by bcoe** [comment]
@proppy awesome, thank you for the contribution (and ignore my comments on #1806).

File: src/apis/homegraph/index.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*! THIS FILE IS AUTO-GENERATED */
+
+import {
+  AuthPlus, 
+  getAPI,
```
Comment (bcoe) [inline src/apis/homegraph/index.ts]: the linting looks a bit weird, i think you might need to run:

```
npm run fix
```

To fix the linting.

File: src/apis/homegraph/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "docs": "typedoc --out docs/",
+    "webpack": "webpack"
+  },
+  "dependencies": {
+    "googleapis-common": "^2.0.4"
+  },
+  "devDependencies": {
+    "gts": "^1.0.0",
+    "null-loader": "^3.0.0",
+    "ts-loader": "^6.0.4",
```
Comment (bcoe) [inline src/apis/homegraph/package.json]: I don't believe that the generator generates with up-to-date dependencies, so you might want to update these accordingly.

**Comment by bcoe** [comment]
Closing in favor of https://github.com/googleapis/google-api-nodejs-client/pull/1814, @JustinBeckwith just ran the whole generator.

We'll make an effort to have this released for you shortly.

**Comment by proppy** [comment]
@bcoe Thanks! and sorry for the lag ;)

**Pull request closed without merging**